src/app.rs

```diff
@@ -864,6 +866,21 @@ impl<T: Frontend> App<T> {
         )
     }
 
+    fn open_copy_file_prompt(&mut self, path: CanonicalizedPath) -> anyhow::Result<()> {
+        self.open_prompt(
+            PromptConfig {
+                title: "Copy path".to_string(),
```

Comment:
I think it would be clearer if the title is formatted like this:

`Copy <current file path> to:`

Reply:
Do you mean put in the actual text of the file being copied?

Reply:
Looking at the current menu, options are:

* a -> Add File (or postfix with / for folder)
* d -> Delete path
* m -> Move path

They do not display anything about the current path.